core/messages/navbar/en.ts

```diff
@@ -8,6 +8,7 @@ export const navbarMessages = i18n('navbar', {
   feeds: 'Manage Feeds',
   feedsByCategory: 'By Category',
   interface: 'User Interface',
+  loading: 'Loading',
```

Comment:
Let’s re-use it from common messages: https://github.com/hplush/slowreader/blob/main/core/messages/common/en.ts#L7